doc/guide/startup.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+	"http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<chapter id="startup">
+  <title>Start up</title>
```

Comment:
We could say 'Cockpit start up' but it sounds just so redundant. The same could be said for any section in the guide, "SSL/TLS Usage" "TCP Port and Address" etc...